charts/multicluster-controlplane/templates/secret.yaml

```diff
@@ -9,30 +9,30 @@ stringData:
     apiserver:
       externalHostname: {{ .Values.apiserver.externalHostname }}
       port: 443
-      {{- if and .Values.apiserver.caFile .Values.apiserver.caKeyFile }}
-      caFile: "/controlplane_config/{{ .Values.apiserver.caFile | base }}"
-      caKeyFile: "/controlplane_config/{{ .Values.apiserver.caKeyFile | base }}"
+      {{- if (and .Values.apiserver.ca .Values.apiserver.cakey) }}
+      caFile: "/controlplane_config/apiserver_ca.crt"
+      caKeyFile: "/controlplane_config/apiserver_ca.key"
       {{- end }}
     etcd:
       mode: {{ .Values.etcd.mode }}
       prefix: {{ .Release.Namespace }}
-      {{- if and (eq .Values.etcd.mode "external") (and .Values.etcd.caFile (and .Values.etcd.certFile (and .Values.etcd.keyFile (not (empty .Values.etcd.servers))))) }}
+      {{- if (and (eq .Values.etcd.mode "external") (and (and .Values.etcd.ca  (and .Values.etcd.cert .Values.etcd.certkey)) (not (empty .Values.etcd.servers)))) }}
```

Comment:
to keep simple, can we just use `{{- if (eq .Values.etcd.mode "external")}}` here?